lib/LaTeXML/Engine/TeX_Paragraph.pool.ltxml

```diff
@@ -68,6 +68,17 @@ DefConstructorI('\noindent', undef, sub {
     # Otherwise ignore.
     return; });
 
+sub alignLine {
+  my ($document, $line, $alignment) = @_;
+  if ($document->isOpenable('ltx:p')) {
+    $document->insertElement('ltx:p', $line, class => 'ltx_align_' . $alignment); }
+  elsif ($document->isOpenable('ltx:text')) {
+    $document->insertElement('ltx:text', $line, class => 'ltx_align_' . $alignment);
+    $document->insertElement('ltx:break'); }
+  else {
+    $document->absorb($line); }
```

Comment:
should this case emit an Info or Warning message? Silently absorbing while dropping the alignment may be worth logging.

Reply:
not a bad idea...